graphql-api needs graphql-tools

```
Error: Cannot find module 'graphql-tools'
    at Object.<anonymous> (.../@accounts/graphql-api/src/utils/authenticated-directive.ts:2:1)
```

Either as peer or direct dep:

https://github.com/accounts-js/accounts/blob/master/packages/graphql-api/package.json

--- a/packages/graphql-api/package.json
+++ b/packages/graphql-api/package.json
@@ -35,6 +35,9 @@
     "nock": "9.0.2",
     "ts-node": "7.0.0"
   },
+  "peerDependencies": {
+    "graphql-tools": "^3.0.0"
+  },
   "dependencies": {
     "@accounts/password": "^0.3.0-beta.21",
     "@accounts/server": "^0.3.0-beta.21",

